File: assignedproblems/planets.java
import java.util.Scanner;
public class planets{
    public static void main( String[] args ) {
        Scanner keyboard = new Scanner(System.in);
        int weight, planet;
        System.out.println("What is your weight on earth in pounds?");
        weight = keyboard.nextInt();
        System.out.println("Out of the following planets, type the corresponding numbe to the planet you want your weight in. \n1:Venus    2:Mars    3:Jupiter    4:Saturn    5:Uranus    6:Neptune \nChosen Planet:");
        planet = keyboard.nextInt();
        if (planet == 1) {
            System.out.println(weight * 0.78);
        }
        else if (planet == 2) {
            System.out.println(weight * 0.39);
        }
        else if (planet == 3) {
            System.out.println(weight * 2.65);
        }
        else if (planet == 4) {
            System.out.println(weight * 1.17);
        }
        else if (planet == 5) {
            System.out.println(weight * 1.05);
        }
        else if (planet == 5) {
            System.out.println(weight * 1.23);
        }
        System.out.println("Next time use the metric system and we won't have this problem.");
    }
}

File: assignedproblems/planets.py
mass = int(input("what is your weight in pounds on Earth? "))
planet = int(input("I have information for the following planets. Enter the number corresponding to the planet you wish to know your weight on: \n 1:Venus    2:Mars    3:Jupiter    4:Saturn    5:Uranus    6:Neptune \nChosen Planet: "))
if planet == 1:
    print(mass * 0.78)
elif planet == 2:
    print(mass * 0.39)
elif planet == 3:
    print(mass * 2.65)
elif planet == 4:
    print(1.17 * mass)
elif planet == 5:
    print(mass * 1.05)
elif planet == 6:
    print(mass * 1.23)
print("Next time use the metric system, and we won't have this problem.")